apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Tests/FilterTest.cs

```diff
@@ -281,6 +289,26 @@ public void Where_CompositeFilter()
         Assert.Equal(expected, query.ToStructuredQuery());
     }
 
+    [Fact]
+    public void NestedCompositeQuery()
+    {
+        var query = GetEmptyQuery()
+            .Where(Filter.Or(Filter.GreaterThanOrEqualTo("a", "x"),
+                              Filter.And(Filter.EqualTo("b", "y"),
+                                        Filter.EqualTo("c", "z"))
+                              )
+                   );
+        var expected = new StructuredQuery
```

Comment:
This is very hard to read at the moment - at least in GitHub. Please could you reformat it? I suspect if you break after `new[]` you'll be able to outdent by quite a lot. You could also consider creating the array elements in local variables before creating `expected`.